scss/components/_welcome.scss

```diff
@@ -0,0 +1,225 @@
+@use '../abstracts' as *;
+
+.welcome {
+    position: relative;
+    @include flexbox(column, center, center);
+    margin: 0 0 2rem 0;
+    text-align: center;
+    width: 100%;
+
+    &__box, &__description {
+        padding: 1rem;
+    }
+
+    &__box {
+        @include flexbox(column, center, center);
+        margin-top: 5rem;
+    }
+
+    &__title {
+        font-size: 3rem;
+        width: 75%;
+    }
+
+    &__geeks-span {
+        color: $primary-color;
+    }
+
+    &__description {
+        margin-top: 1rem;
+        font-size: 1.5rem;
+    }
+
+
+    &__btn-box {
+        margin: 1.5rem;
+    }
+
+    &__button {
+        margin: 0 0.5rem;
+        padding: 1.5rem 2.5rem;
+        cursor: pointer;
+
+        &--primary {
+            @include btn-primary();
+        }
+
+        &--secondary {
+            @include btn-secondary();
+        } 
+    }
+
+    &__cart-icon {
+        padding-right: 0.5rem;
+    }
+
+    &__preview-icon {
+        margin-left: 0.5rem;
+    }
+
+    &__rating {
+        @include flexbox();
+        margin: 1.5rem auto;
+    
+        &-score, &-description {
+            margin: 0.2rem;
+            padding: 0.5rem 1rem;
+        }
+
+        &-score {
+            font-size: 1.1rem;
+            border: 0.1rem solid $border-color;
+            border-radius: 0.3rem;
+        }
+
+        &-description {
+            font-size: 1.5rem;
+        }
+
+        &-stats {
+            font-weight: 700;
+        }
+    }
+
+    &__star-icon {
+        font-size: 1.1rem;
+        margin-right: 0.5rem;
+        color: $star-color;
+    }
+
+    &__features {
+        margin: 5rem;
+        padding: 2rem;
+        width: 100%;
+        border-bottom: 1px solid $border-bottom-color;
+        &-box {
+            margin: 2rem 0;
+            @include flexbox(row, center, flex-start);
+        }
+
+        &-icon-box {
+            @include flexbox(row, center, center);
+            background-color: $primary-light;
+            min-width: 4.8rem;
+            height: 4.8rem;
+            flex-shrink: 0;
+            border-radius: 1rem;
+        }
+
+        &-icon {
+            font-size: 2rem;
+            color: $primary-dark;
+            z-index: 1;
+        }
+
+        &-info {
+            margin-left: 1.5rem;
+            text-align: left;
+            flex-grow: 1;
+        }
+
+        &-title, &-description {
+            padding: 0.5rem;
+        }
+
+        &-title {
+            font-size: 1.6rem;
+            font-weight: 600;
+        }
+
+        &-description {
+            font-size: 1.5rem;
+            width: 80%;
+            color: $secondary-btn-color;
+        }
+    }
+}
```

Comment:
I analyzed the structure and that's true - this section seems much bigger than others. Two things come to my mind:
1.  Regarding BEM elements - f.e.. having the div `features-info` with heading `title` and paragraph `description` in it, I could adjust it to be `features-info-title / description` - and similarly in other parts - in this case I would save some space and improve code readability.
2. Divide this section for two - code part starting from `welcome__features` could be treated as different section, and styled in different `.scss` file.

Since option 2 demands pretty big html structure refactor, I would more agree for splitting to few BEM elements, but wanted to know what do you think about the second idea? Please, let me know if I understood your suggestion with splitting correctly.